e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -17,6 +17,13 @@ param(
     [Parameter()]
     [switch] $InstallDependencies,
 
+    # Obsoleted. Recommend docker on WSL2 instead - setup with <repo>/e2e/test/docker/haproxy.sh on WSL.
+    # Installing Docker Desktop on Windows require using HyperV or WSL, with HyperV by default, which can interfere with certain proxy setup for testing.
+    # USE THIS OPTION IF YOU KNOW YOU NEED THIS.
+    # Specify this on the first execution to get everything installed in powershell. It does not need to be run every time.
+    [Parameter()]
+    [switch] $InstallDockerDesktopOnWindows,
```

Comment:
Why not include with $InstallDependencies?

Reply:
By default, we should not use docker for Windows, current version (v20.x) have broken virtual network feature.